### DISCUSSION TITLE
make vcf error 126

### DISCUSSION THREAD

**Discussion by goddy93** [biostars/biostar-handbook-issues#223] [Q&A]
Hi,

I tried the command "make vcf". 

make vcf
mkdir -p reads
fastq-dump -F -X 10000 --split-files -O reads SRR1553425
bash: /Users/me/bin/sratoolkit.3.0.0-ubuntu64/bin/fastq-dump: cannot execute binary file
make: *** [Makefile:110: reads/SRR1553425_1.fastq] Error 126

**Comment by ialbert** [comment]
it appears that you have downloaded the wrong version of the fastq-dump, probably you have downloaded the Linux version instead of the MacOS version.
